lib/services/consumed/status-bar/signal-list-view.js

```diff
@@ -17,15 +17,15 @@ const wsKernelCommands = [
 ];
 
 export default class SignalListView {
-  onConfirmed: ?(command: { command: string }) => void;
   panel: ?atom$Panel;
   previouslyFocusedElement: ?HTMLElement;
   selectListView: SelectListView;
-  kernel: ?Kernel;
+  store: ?Store;
+  handleKernelCommand: Function;
 
-  constructor(kernel: Kernel) {
-    this.kernel = kernel;
-    this.onConfirmed = null;
+  constructor(store: Store, handleKernelCommand: Function) {
+    this.store = store;
+    this.handleKernelCommand = handleKernelCommand;
```

Comment:
you must declare this field outside the constructor and give it a flow type before you can use it.